src/platform/Linux/CHIPPlatformConfig.h

```diff
@@ -27,6 +27,12 @@
 
 #define CHIP_CONFIG_ABORT() abort()
 
+extern "C" int __cxa_atexit(void (*func)(void *), void * arg, void * d);
+#define CHIP_CXA_ATEXIT(f, p) __cxa_atexit((f), (p), nullptr)
+
+#define CHIP_CONFIG_DYNAMICALLY_ALLOCATED_GLOBALS 1
```

Comment:
Is this just for CI or you're going to merge this? If the latter, I'm not sure it's a good idea to set this unconditionally for Linux. Perhaps, adding a GN flag to enable this or at least defining these conditionally using `#ifndef` would help Linux device makers to select the desired configuration.

Reply:
OK, I'll revert this for now. GN is new to me, so I would appreciate any ideas about how I could go about enabling this just for CI.